addons/im_livechat/static/src/embed/common/close_confirmation.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.CloseConfirmation">
+        <div
+            class="o-livechat-CloseConfirmation z-1 bg-black-50 position-absolute w-100 h-100 d-flex justify-content-center align-items-center"
+            t-on-keydown.capture.stop="onKeydown"
+            t-on-click.stop="onClick"
+            t-ref="closeConfirmation"
+            tabindex="1"
+        >
+            <div class="o-livechat-CloseConfirmation-dialog rounded bg-white bg-opacity-100 p-3 m-3 d-flex flex-column">
+                <div class="d-flex justify-content-end">
+                    <button class="o-livechat-CloseConfirmation-close btn-close" />
+                </div>
+                <span class="pt-2 pb-2">You're about to leave the conversation, proceed?</span>
+                <button class="o-livechat-CloseConfirmation-leave btn btn-primary p-2" title="Leave conversation">Yes, leave conversation</button>
```

Comment:
I would prefer `btn-danger`, this should scare the user a bit. Primary on the other-hand encourages the user to click without thinking twice.

I would also put `fa-sign-out` icon, because an icon gives a better grasp of what clicking on the button will result too.